plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineOptionsSelectionDialog.java

```diff
@@ -140,7 +140,7 @@ public String getText(Object element) {
         return null;
       }
       String simpleName = Signature.getSimpleName(element.toString());
-      return String.format("%s - %s", simpleName, element.toString());
+      return String.format("%s - %s", simpleName, element.toString()); //$NON-NLS-1$
```

Comment:
It's used in an _Open Type_-like dialog to search for PipelineOptions classes. `pipelineoptions.search.typenames`?

<img width="609" alt="screen shot 2017-08-09 at 4 14 16 pm" src="https://user-images.githubusercontent.com/202851/29141884-0766554a-7d1e-11e7-9583-b846a9258810.PNG">